### DISCUSSION TITLE
Bus 'ide.0' not found

### DISCUSSION THREAD

**Discussion by NuePlusPlus** [utmapp/UTM#3964] [UTM Help (macOS)] [answered]
<img width="272" alt="Screen Shot 2022-05-01 at 5 28 16 PM" src="https://user-images.githubusercontent.com/104742066/166171352-b3297380-19aa-43b8-b5d9-52d3d061edb2.png">

**Comment by osy** [comment, marked as answer]
IDE is not supported on aarch64. Use another interface such as virtio or USB or NVME
